src/pages/image/_id.vue

```diff
@@ -45,7 +45,7 @@
           v-if="image.creator"
           path="image-details.creator"
           tag="span"
-          class="font-semibold leading-[130%]"
+          class="font-semibold leading-[130%] text-base"
```

Comment:
```suggestion
          class="leading-snug"
```
Could you add `font-semibold text-base` classes to the outer span (line 40), so that we don't have to repeat it for all children? We can then remove `text-base font-semibold` from the `h1` with image title. It would also be nice to replace the outer span (line 40) with a `div` because you are not supposed to have block element inside a `span`.